cmd/main.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2021 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"io"
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/klog"
+
+	"github.com/cilium/cilium-sysdump/internal/defaults"
+	"github.com/cilium/cilium-sysdump/pkg/sysdump"
+)
+
+func main() {
+	fs := flag.NewFlagSet("", flag.ExitOnError)
```

Comment:
probably makes sense to use cobra/pflag for parsing command line to be consistent with cilium-cli etc 👀